cron/mozdefStateDefaultMappingTemplate.json

```diff
@@ -0,0 +1,387 @@
+{
+    "order": 0,
+    "template": "mozdefstate",
+    "settings": {
+        "number_of_shards": 1,
+        "number_of_replicas": 0
+    },
+    "mappings": {
+        "mozdefstats": {
+            "_all": {
+                "enabled": true
+            },
+            "dynamic_templates": [{
+                    "string_fields": {
+                        "match": "*",
+                        "match_mapping_type": "string",
+                        "mapping": {
+                            "type": "keyword"
+                        }
+                    }
+                },
+                {
+                    "double_fields": {
+                        "match": "*",
+                        "match_mapping_type": "double",
+                        "mapping": {
+                            "type": "keyword"
+                        }
+                    }
+                },
+                {
+                    "long_fields": {
+                        "match": "*",
+                        "match_mapping_type": "long",
+                        "mapping": {
+                            "type": "keyword"
+                        }
+                    }
+                }
+            ],
+            "properties": {
```

Comment:
Since most of these fields aren't used in the state index, can we remove the unnecessary ones?